srcalc/src/main/java/gov/va/med/srcalc/domain/Calculation.java

```diff
@@ -162,19 +164,33 @@ void setValues(final SortedSet<Value> values)
     }
     
     /**
+     * Returns an unmodifiable view of the last-calculated outcomes.
+     */
+    public SortedMap<String, Double> getOutcomes()
```

Comment:
The assumption is fine as long as fOutcomes is only set in calculate-esque methods so that the map can be controlled.